pkgs/development/libraries/fltk/default.nix

```diff
@@ -27,20 +17,33 @@ stdenv.mkDerivation {
         then [ freetype libtiff ]
         else [ xlibsWrapper libXi freeglut ]);
 
-  configureFlags = [
-    "--enable-gl"
-    "--enable-largefile"
-    "--enable-shared"
-    "--enable-threads"
-    "--enable-xft"
+  cmakeFlags = [
+    "-DOPTION_BUILD_SHARED_LIBS=1"
+    "-DOpenGL_GL_PREFERENCE=GLVND"
   ];
 
-  enableParallelBuilding = true;
-
   meta = with lib; {
     description = "A C++ cross-platform lightweight GUI library";
     homepage = "https://www.fltk.org";
     platforms = platforms.linux ++ platforms.darwin;
     license = licenses.gpl2;
```

Comment:
gpl2Only or gpl2Plus?

Reply:
Actually not GPL at all, but how is this related to the change?

Reply:
Than we can fix it while we are at it.

> how is this related to the change?

Not at all but I just noticed it so we can fix it.

Reply:
Added this change to the PR, but I prefer to keep changes with different topics in seperate PRs.

Reply:
Same package is topic enough for me and doing a PR just to update a license is pretty minor. Also in the time you discussed with me would have been enough to update the license.